x-pack/metricbeat/module/airflow/_meta/README.md

```diff
@@ -0,0 +1,11 @@
+# Airflow metricbeat module
+
+## How to test
+You don't need a running Airflow instance in order to test the module:
```

Comment:
It may still worth to add short instructions about how the module could be tested with Airflow, for future developers working on this module.